RiotSwiftUI/Modules/Room/LocationSharing/View/UserLocationAnnotatonView.swift

```diff
@@ -38,7 +38,14 @@ class UserLocationAnnotatonView: MGLUserLocationAnnotationView {
         // TODO: Use a reuseIdentifier
         super.init(annotation: userLocationAnnotation, reuseIdentifier: nil)
         
-        self.addUserMarkerView(with: userLocationAnnotation.avatarData)
+        switch userLocationAnnotation.coordinateType {
+        case .user:
+            self.addUserMarkerView(with: userLocationAnnotation.avatarData)
+        case .pin, .generic:
```

Comment:
Can be interesting to create a `PinLocationAnnotation` and `init(pinLocationAnnotation: PinLocationAnnotation)`